app/[locale]/[...slug]/page.tsx

```diff
@@ -97,8 +98,7 @@ export default async function Page({
         lastEditLocaleTimestamp={lastEditLocaleTimestamp}
         contentNotTranslated={!isTranslated}
         contributors={contributors}
-        // TODO: Remove this once we have a real timeToRead value
-        timeToRead={2}
+        timeToRead={Math.round(timeToRead.minutes)}
```

Comment:
might need to add a default value here as a fallback in case `timeToRead.minutes` returns something undefined